DRAFT.md

```diff
@@ -983,8 +990,8 @@ constexpr void swap(indirect& other) noexcept(
   || allocator_traits::is_always_equal::value);
```

Comment:
```suggestion
  || allocator_traits<Allocator>::is_always_equal::value);
```

And on the line above, which I can't suggest a change for because Github thinks it's not part of the review.